kafka/consumer-group/consumer_group.go

```diff
@@ -241,16 +245,23 @@ func (cg *ConsumerGroup) InstanceRegistered() (bool, error) {
 	return cg.instance.Registered()
 }
 
-func (cg *ConsumerGroup) CommitUpto(message *sarama.ConsumerMessage) error {
-	cg.offsetManager.MarkAsProcessed(message.Topic, message.Partition, message.Offset)
+func (cg *ConsumerGroup) CommitUpto(message *sarama.ConsumerMessage, consumerGroupName string, enabled bool) error {
```

Comment:
let's always be explicit about naming variables

```suggestion
func (cg *ConsumerGroup) CommitUpto(message *sarama.ConsumerMessage, consumerGroupName string, trackingEnabled bool) error {
```